roles/cluster_api/tasks/patch.yml

```diff
@@ -1,16 +1,5 @@
-  # Copyright (c) 2023 VEXXHOST, Inc.
-  #
-  # Licensed under the Apache License, Version 2.0 (the "License"); you may
-  # not use this file except in compliance with the License. You may obtain
-  # a copy of the License at
-  #
-  #      http://www.apache.org/licenses/LICENSE-2.0
-  #
-  # Unless required by applicable law or agreed to in writing, software
-  # distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-  # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-  # License for the specific language governing permissions and limitations
-  # under the License.
+# Copyright (c) 2025 VEXXHOST, Inc.
+# SPDX-License-Identifier: Apache-2.0
 
 - name: Set node selector for Cluster API components
```

Comment:
Small question, could we not use the same approach here to update the environment variables?

Reply:
We can pass this as part of `_cluster_api_environment` but then this option will be applied during bootstrap or upgrade only(right now we use` _cluster_api_environment` for sane global defaults safe for ANY perovider not only openstack). I think we need way to pass this also for existing installations or when there is a need to reconfigure this at any time.